composer.json

```diff
@@ -13,8 +13,8 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-config": "2.2.3",
-        "zendframework/zend-di": "2.2.3"
+        "zendframework/zend-config": "~2.3",
```

Comment:
this might actually also be backwards incompatible. we need a version constraint that will allow all versions between 2.2.3 (inclusive) and 3.0.0 (not inclusive). Sorry I overlooked it at first. You can see into fixing this or I will make sure we update this internally and push to github.